tmuxinator.gemspec

```diff
@@ -42,5 +42,14 @@ Gem::Specification.new do |s|
   s.add_dependency "erubis", "~> 2.6"
 
   s.add_development_dependency "bundler", "~> 1.3"
+  s.add_development_dependency "rake", "~> 10.4"
+  s.add_development_dependency "rspec", "~> 3.3"
+  s.add_development_dependency "simplecov", "~> 0.11.0"
+  s.add_development_dependency "coveralls", "~> 0.7"
+  s.add_development_dependency "awesome_print", "~> 1.2"
+  s.add_development_dependency "pry", "~> 0.10"
+  s.add_development_dependency "factory_girl", "~> 4.5"
+  s.add_development_dependency "activesupport", "< 5.0.0"  # Please see issue #432
```

Comment:
I think I have everything updated now.

* Rebased my branch against master
* Updated CHANGELOG
* All gem deps moved to tmuxinator.gemspec (including rubocop)
* bumped required Ruby version to 2.0.0 in tmuxinator.gemspec
* tmux 2.2 and 2.3 added to TravisCI test matrix